config/jobs/image-pushing/k8s-staging-test-infra.yaml

```diff
@@ -273,3 +273,27 @@ postsubmits:
           - --project=k8s-staging-test-infra
           - --env-passthrough=PULL_BASE_REF
           - triage/
+    -name: post-test-infra-push-pytest
```

Comment:
test failed, add a space between `-` and `name`

Reply:
Fixed!